<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propertyEditorDialog</class>
 <widget class="QDialog" name="propertyEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Sidekick Property editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Suggested name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="filterColumnsLineEdit">
          <property name="placeholderText">
           <string>filter files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="locationCheckBox">
          <property name="text">
           <string>Location</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cameraCheckBox">
          <property name="text">
           <string>Camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="artistCheckBox">
          <property name="text">
           <string>Artist</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="APropertyTreeView" name="propertyTreeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update properties</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="updateProgressBar">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="text">
        <string>Rename to suggested names</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="geocodingGroupBox">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Geocoding</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="AGeocoding" name="geocodingWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Map</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QQuickWidget" name="mapQuickWidget">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
           </font>
          </property>
          <property name="resizeMode">
           <enum>QQuickWidget::SizeRootObjectToView</enum>
          </property>
          <property name="source">
           <url>
            <string>qrc:/amapviewer.qml</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="statusBarLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>APropertyTreeView</class>
   <extends>QTreeView</extends>
   <header>apropertytreeview.h</header>
  </customwidget>
  <customwidget>
   <class>AGeocoding</class>
   <extends>QWidget</extends>
   <header>ageocoding.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
